caas/kubernetes/provider/k8s.go

```diff
@@ -401,7 +401,7 @@ func (k *kubernetesClient) EnsureService(
 	logger.Debugf("creating/updating application %s", appName)
 
 	if numUnits < 0 {
-		return errors.Errorf("number of units must be >= 0")
+		return errors.Errorf("number of units must be > 0")
```

Comment:
Above check should be <= 0 now

Reply:
Also, the if numUnits > 0 below can disappear

Reply:
ah right